src/app/api/analytics/background_task.py

```diff
@@ -105,12 +112,31 @@ def run():
     app.api.analytics.storage.global_storage[_COLLECTION_COUNT] = asyncio.run(
         collection_counts(COLLECTION_ROOT_ID, AggregationMappings.lrt)
     )
+    app.api.analytics.storage.global_storage[_COLLECTION_COUNT_OER] = asyncio.run(
+        collection_counts(COLLECTION_ROOT_ID, AggregationMappings.lrt, oer_only=True)
+    )
 
-    all_collections = asyncio.run(get_ids_to_iterate(node_id=COLLECTION_ROOT_ID))
+    all_collections = [
+        Row(id=value["value"], title=key)
```

Comment:
Aah, good spotting. This could resolve the string issue. I check that now.

Reply:
Ay, you are right - the issue is that I use str in Collection class - I added a todo there. Refactoring it now might blow up this PR, since that class is used a lot.